From 71ebb64f7560f297226e61aa68b867c308ace05c Mon Sep 17 00:00:00 2001
Message-Id: <71ebb64f7560f297226e61aa68b867c308ace05c.1526826081.git.pavel-rehak@email.cz>
From: Pavel Rehak <pavel-rehak@email.cz>
Date: Sun, 20 May 2018 16:10:41 +0200
Subject: [PATCH] Inverted commas change an array to a variable.

---
 bash-ini-parser | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bash-ini-parser b/bash-ini-parser
index 23384e0..c21c902 100755
--- a/bash-ini-parser
+++ b/bash-ini-parser
@@ -111,7 +111,7 @@ function cfg_writer {
             echo $var=\"${!var}\" #output var
          else 
             echo ";$var is an array" # add comment denoting var is an array
-            eval 'echo $var=\"${'$var'[*]}\"' # output array var
+            eval 'echo $var=${'$var'[*]}' # output array var
          fi
       done
    done
-- 
2.17.0

