From c92c863f5cb43a37dc66e39cf2f97f5800562b21 Mon Sep 17 00:00:00 2001
Message-Id: <c92c863f5cb43a37dc66e39cf2f97f5800562b21.1526826150.git.pavel-rehak@email.cz>
From: Pavel Rehak <pavel-rehak@email.cz>
Date: Sun, 20 May 2018 16:16:00 +0200
Subject: [PATCH] Delete ` and $ from variables. Ban to execute a malicious
 code.

---
 bash-ini-parser | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/bash-ini-parser b/bash-ini-parser
old mode 100755
new mode 100644
index 23384e0..ac7049e
--- a/bash-ini-parser
+++ b/bash-ini-parser
@@ -31,6 +31,8 @@ function cfg_parser {
    debug "escaped ["
    ini="${ini//]/\\]}"
    debug "escaped ]"
+   ini="${ini//\$/}"            # remove executable
+   ini="${ini//\`/}"            # remove executable
    IFS=$'\n' && ini=( ${ini} )  # convert to line-array
    debug
    ini=( ${ini[*]/#*([[:space:]]);*/} )
-- 
2.17.0

